docs/input-manifest/example.yaml

```diff
@@ -268,8 +345,6 @@ loadBalancers:
       dns:
         dns_zone: dns-zone
         provider: cloudflare-1
-        # this hostname will be used for DNS records
-        # keep in mind that DNS zone will be included
```

Comment:
Do we want to remove the comments from the manifest example?

Reply:
I moved the comments from being all over the place to somewhere where it made more sense. In this case, they can be found [here](https://github.com/Berops/claudie/blob/feat/update-docs/docs/input-manifest/example.yaml#L320).

Reply:
Thanks, I haven't realized that.